### DISCUSSION TITLE
GWAT

### DISCUSSION THREAD

**Discussion by Saman-M-B** [fabern/LWFBrook90.jl#90] [Q&A]
Hello,
I'm simulating a case study of a crop field with LWFBrook90.jl model but there are some issues with the results.
First, the calculated GWAT doesn't make sense because even the soil is almost dry and there is no precipitation, there are high amounts of GWAT. I was wondering how it calculates GWAT?
Second, in LWFBrook90R there is a representation of soil type percentages as input values but not in LWFBrook90.jl. How does the model distinguish between different soil types other than hydraulic conductivity values?
Third, I used the command "get_water_partitioning" to get the values of ETa in particular but also that doesn't make sense where for the whole period of almost one year, the estimated ETa is around 200 mm which for a cropping season is too low. Is there any other criteria maybe assumed that this case of crop land wouldn't make a good result?
Thanks

**Comment by fabern** [comment]
Hi,

1) GWAT is the groundwater storage. So it depends on its initial condition. Then it is filled by the water that drains at the bottom of the soil layer, and emptied by the water computed as GWFL and . See flow chart in the [documentation](https://fabern.github.io/LWFBrook90.jl/stable/model/#Detailed-description) and if something is missing there is always the documentation of the base model: http://www.ecoshift.net/brook/b90doc.html

2) I don't know what percentages you refer to from the input of LWFBrook90R. Generally, the model only uses the values of the van Genuchten parameters to distinguish different soil layers as specified in `soil_horizons.csv`, please see: https://fabern.github.io/LWFBrook90.jl/stable/user-guide/#Structure-of-input-data

3) This is difficult to diagnose without the specific simulation. There are many options specified and it requires some understanding of the underlying assumptions to translate the default forest parameters to croplands. In principle one should also be able to simulate croplands with LWFBrook90.jl. You mention that you expect higher actual evapotranspiration ETa. However, you also mention that you have no rainfall and very dry soil. So given these information it makes sense to me that evapotranspiration is low.

**Comment by Saman-M-B** [comment]
Thanks for your reply. To better clarify my questions, i provided examples of my test simulations.
so the picture below is for a vegetative period test run. the initial conditions are 0 and the lysimeter data shows no seepage but the model is reaching a high values of GWAT even before precipitation event and it seems is almost not responsive of the rain events after that. so it seems one possibility as if the soil contains no water and everything goes directly to groundwater storage.
![image](https://github.com/fabern/LWFBrook90.jl/assets/156809382/5269d23f-5510-4188-bfdf-cfd7a735315d)
![image](https://github.com/fabern/LWFBrook90.jl/assets/156809382/fd647613-1cef-4777-8a0d-b9509825b6c7)
![image](https://github.com/fabern/LWFBrook90.jl/assets/156809382/75706b6b-e47f-4011-95fd-dfc5e4955c1c)

**Comment by fabern** [comment]
From a water balance perspective your simulations make perfectly sense. In an initial phase until May 2019 your GWAT increases because there is freely draining water from your wet soil ("VRFL(N)" in below schematic). GWAT increases to ~180mm, while total SWAT decreases  by this amount. The initial condition you give to your soil matric potential is wet enough that there is freely draining water (even without any precipitation event), which in turn fills the groundwater storage GWAT. 

In a second phase GWAT stabilizes at a more or less constant level. But this is a dynamic equilibrium between the refill it is still receiving from the soil layer VRFL(N) and the outflow of the GWAT (linear reservoir) as GWFL and SEEP. You can see in your fourth plot panel that these are similar. The cumulative "Outflow SWATI" shows the sum of all flows leaving the SWATI compartments, which in this second phase evolves parallel to the "Outflow Model" which shows the sum of all flows leaving the whole model domain (i.e. EVAP, SEEP, FLOW).

If the free drainage in the first phase is unexpected (since you appear to have lysimeter data) you should check your initial conditions and the soil hydraulic parameters.

![](https://fabern.github.io/LWFBrook90.jl/stable/assets/b90flow.gif)

**Comment by Saman-M-B** [comment]
Thanks for your response. Is the GWAT that the model reports cumulative?
Also, I have a question regarding what you said. their initial conditions are 0 for all parameters so I'm not sure what you mean by "wet soil" and "The initial condition you give to your soil matric potential is wet enough that there is freely draining water (even without any precipitation event), which in turn fills the groundwater storage GWAT." I don't know where we could implement this initial condition of soil matric potential to say that the soil's initial condition should be dry clearly.

**Comment by fabern** [comment]
GWAT is a state variable and it represents a storage. Thus, yes, the change in storage considers cumulative in- and outflows since the beginning of the simulation. This is the case for all the storages (green boxes in above schematic).

The initial wetness of the soil at the beginning of the simulation is given as initial condition in `soil_discretization.csv`. You can try to change this to a more negative value. The volume of water that is freely draining depends on your soil properties. Frequently on assumes a soil matric potential of -33 kPa to correspond to field capacity (e.g. https://www.nature.com/scitable/knowledge/library/soil-water-dynamics-103089121/)

**Comment by Saman-M-B** [comment]
I was also wondering if there is a function to retrieve the values of storage but in different depths such as GWAT or since the model considers these as storage, you can only get the final values? Cause I think the LWFBrook90R model provides these values per soil layer for each timestep.
